### PR TITLE
Updates to ISV submission guide

### PR DESCRIPTION
**Pull request by IanHakes-okta**
## Description:
- **What's changed?**
Changes to clean up the ISV submission guide and reduce the redundant and unneeded FAQs from the provisioning section.
- **Is this PR related to a Monolith release?**
No
### Resolves:

* [OKTA-262042](https://oktainc.atlassian.net/browse/OKTA-262042)
* [OKTA-272471](https://oktainc.atlassian.net/browse/OKTA-272471)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/submit-app/faqs/index.md [hunk 1]
```diff
@@ -2,138 +2,42 @@
...
-
-**My app currently supports WS-FED for single sign-on. Can I use the App Wizard?**
-
-The Okta App Wizard only supports SAML 2.0 for federated single sign-on. If your app supports WS-Fed, create a [WS-Fed Template App](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Configuring_WS_Federation). Afterwards, the template application you created can be used only within your account. In order to promote your template app to the Okta Integration Network, please email a screenshot with the details of the configured app to <developers@okta.com>, with your app name in the subject line.
-
-**I am creating a SWA app using the App Wizard but my application has fields on the login page in addition to the standard username and password (for example: CustomerID, OrgID). Can an app with additional fields on the login be configured using the App Wizard?**
+**Q: I am creating a SWA app using the App Wizard but my application has fields on the login page in addition to the standard username and password (for example: CustomerID, OrgID). Can an app with additional fields on the login be configured using the App Wizard?**
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app/faqs/index.md]: I am creating a SWA app using the App Wizard but my application has fields on the **sign-in** page in addition to the standard username and password (for example: CustomerID, OrgID). Can an app with additional fields on the **sign-in page** be configured using the App Wizard?

File: packages/@okta/vuepress-site/docs/guides/submit-app/overview/index.md [hunk 2]
```diff
@@ -2,18 +2,21 @@
...
 ---
 
-This guide outlines the process by which independent software vendors (ISVs), Okta customers, and IT system integrators  can submit an app for review by Okta with the goal of publishing it to the Okta Integration Network (OIN).
+This guide outlines the process by which independent software vendors (ISVs), Okta customers, or IT system integrators can submit an app for review by Okta with the goal of publishing it to the [Okta Integration Network](https://www.okta.com/integrations/). Integrating your application with the Okta Integration Network is completely cost free.
 
-For details on how to build a SCIM provisioning app integration, see our [SCIM Provisioning Guide](/docs/guides/build-provisioning-integration/).
+For details on how to build a SCIM provisioning integration, see our [SCIM Provisioning Guide](/docs/guides/build-provisioning-integration/).
 
-For details on how to build a SAML app integration, see our [SAML Guide](/docs/guides/build-sso-integration/).
+For details on how to build a SAML single sign-on integration, see our [SAML Guide](/docs/guides/build-sso-integration/).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app/overview/index.md]: Single Sign-On

File: packages/@okta/vuepress-site/docs/guides/submit-app/submit/index.md [hunk 3]
```diff
@@ -2,24 +2,37 @@
...
+
+Once Okta completes all internal QA testing and confirms that any requested changes have been made, the analysts publish the Okta-Verified integration directly to the OIN catalog.
+
+Have you submitted an app but haven't seen a change in the review status in the [OIN Manager](https://oinmanager.okta.com)? Send an email to <developers@okta.com>.
 
-Once Okta completes all internal QA processes and confirms that any requested changes are complete, the analysts publish the Okta-Verified app to the Okta Integration Network.
+If you have already submitted an integration to the OIN, but need to make changes, submit your application with the changes through the OIN Manager, as if your application were not listed in the OIN. When your submission is reviewed by Okta, we compare your submission against the existing customer-facing version, and update it accordingly. If there are special instructions you'd like to provide to us, add them in the Additional Instructions text box on the submission form, and we will contact you directly with any questions. Your app goes through the Okta QA process every time you add additional attributes.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app/submit/index.md]: If you have already submitted an integration to the OIN, but need to make changes, submit your application with the changes through the OIN Manager as if your application **isn't** listed in the OIN. When your submission is reviewed by Okta, we compare your submission against the existing customer-facing version and update it accordingly. If there are special instructions you'd like to provide to us, add them in the **Additional Instructions** text box on the submission form, and we will contact you directly with any questions. Your app goes through the Okta QA process every time you add additional attributes.

removed the comma before the "and" in the second sentence also.
Bold Additional Instructions and remove the word "text"

Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/submit/index.md]: Changes made.

File: packages/@okta/vuepress-site/docs/guides/submit-app/faqs/index.md [hunk 4]
```diff
@@ -2,138 +2,42 @@
...
-No, integrating your application with the Okta Integration Network is completely free
-
-Paid customers can also utilize all application integrations in the OIN free of charge.
-
-**If I submit my app with a set of attributes, can I then add attributes during the testing phase of the app?**
-
-Yes. You can add attributes to the app you are testing in your dev org. Email <developers@okta.com> with the changes.
-
-**After my app has been published, and I add additional attributes, how do I republish my app? Can I republish frequently?**
+Yes, the session time out default is two hours, but the Okta administrator can customize the default. This session time out is an IdP setting and therefore it is global and applies to all applications in an Okta org. See the "Creating Sign-on Policies and Adding Rules" section in our [Security Policies](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Security_Policies) documentation.
```
[the remark this answers is not in the document]
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/faqs/index.md]: Fixed.

File: packages/@okta/vuepress-site/docs/guides/submit-app/faqs/index.md [hunk 5]
```diff
@@ -2,138 +2,42 @@
...
 
-Yes, you can republish your app, but we recommend you don't do it frequently. Your app goes through the Okta QA process every time you add additional attributes.
+### SAML
 
-**I am an ISV whose app is already listed in the OIN. How do I request changes/updates to the existing integration?**
+**Q: Does Okta support single logout or single sign-out using the SAML protocol?**
```
[the remark this answers is not in the document]
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/faqs/index.md]: Changed to "support Single Logout (SLO)"

File: packages/@okta/vuepress-site/docs/guides/submit-app/faqs/index.md [hunk 6]
```diff
@@ -2,138 +2,42 @@
...
+**Q: My app currently supports WS-FED for single sign-on. Can I use it instead of SAML?**
 
-Yes. One of the key benefits of developing a pre-built integration with Okta is that you can leverage our existing active directory support. By integrating with Okta (for single sign-on and provisioning), you effectively have the ability to integrate with your customer's on-premise AD or LDAP infrastructure for authentication (log into your cloud app with their corporate password), authorization (use details like AD groups to drive access rights), and provisioning policies.
+The Okta App Wizard only supports SAML 2.0 for federated single sign-on. If your app supports WS-Fed, create a [WS-Fed Template App](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Configuring_WS_Federation). Afterwards, the template application you created can be used only within your account. In order to promote your template app to the Okta Integration Network, please email a screenshot with the details of the configured app to <developers@okta.com>, with your app name in the subject line.
```
[the remark this answers is not in the document]
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/faqs/index.md]: "You can only use this created template application within your own account."

File: packages/@okta/vuepress-site/docs/guides/submit-app/faqs/index.md [hunk 7]
```diff
@@ -2,138 +2,42 @@
...
 ---
 
-## OIN Publishing Questions
+## OIN Technical Questions
 
-**I am an ISV and want to integrate with Okta for SAML and SCIM. Do I need to contact Okta first to start integrating my application?**
+**Q: My customer is asking for AD (Active Directory) integration. If I integrate with Okta, can I connect to my customer's on-premise directory?**
 
-We work hard to make it easy for you to integrate with Okta. To that end, you don't have to contact Okta to integrate with Okta. Just start with the following SCIM or SAML documentation links, and you should be all set! Of course, if you do have questions, our developer support team <developers@okta.com> is here to help you.
+Yes. One of the key benefits of developing a pre-built integration with Okta is that you can leverage our existing active directory support. By integrating with Okta (for single sign-on and provisioning), you effectively have the ability to integrate with your customer’s on-premise AD or LDAP infrastructure for authentication (log into your cloud app with their corporate password), authorization (use details like AD groups to drive access rights), and provisioning policies.
```
[the remark this answers is not in the document]
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/faqs/index.md]: I made it into **3** sentences:
"By integrating with Okta (for Single Sign-On or provisioning), you effectively have the ability to integrate with your customer's on-premise AD or LDAP infrastructure for authentication. Your end users can sign in to your cloud app using their corporate password. You also be able to do things like use AD groups to drive access rights for authentication and provisioning policies."

**Pull request merged**